src/web/mjs/connectors/Eclypse.mjs

```diff
@@ -0,0 +1,13 @@
+import WordPressMadara from './templates/WordPressMadara.mjs';
+
+export default class Eclypse extends WordPressMadara {
+
+    constructor() {
+        super();
+        super.id = 'eclypse';
+        super.label = 'eclypse';
+        this.tags = [ 'manga', 'scanlation', 'webtoons', 'french' ];
+        this.url = 'https://eclypse-scan.com';
+    }
+}
+
```

Comment:
Lint ;)

Reply:
I don't understand aha

Reply:
check this ;)
https://github.com/manga-download/hakuneko/pull/4650/files